vignettes/ARTnetIntro.R

```diff
@@ -0,0 +1,185 @@
+## ---- message=FALSE------------------------------------------------------
+library("ARTnet")
+
+## ------------------------------------------------------------------------
+epistats1 <- build_epistats(geog.lvl = "state", geog.cat = "GA")
+
+## ------------------------------------------------------------------------
+epistats2 <- build_epistats(geog.lvl = "state", geog.cat = "GA",
+                           init.hiv.prev = 0.17)
+
+## ------------------------------------------------------------------------
+epistats3 <- build_epistats(geog.lvl = "state", geog.cat = "GA", init.hiv.prev = 0.17)
+
+## ------------------------------------------------------------------------
+epistats4 <- build_epistats(geog.lvl = "division", geog.cat = "5", init.hiv.prev = 0.17)
+epistats5 <- build_epistats(geog.lvl = "region", geog.cat = "3", init.hiv.prev = 0.17)
+
+## ------------------------------------------------------------------------
+epistats6 <- build_epistats(geog.lvl = NULL)
+
+## ------------------------------------------------------------------------
+#Racial stratification
+epistats7 <- build_epistats(geog.lvl = "state", geog.cat = "WA", race = TRUE, 
+                            init.hiv.prev = c(0.17, 0.1, 0.13))
+
+#No racial stratification
+epistats5 <- build_epistats(geog.lvl = "state", geog.cat = "GA", race = FALSE, 
+                            init.hiv.prev = 0.17)
+
+## ------------------------------------------------------------------------
+#Set age limits
+epistats8 <- build_epistats(geog.lvl = "state", geog.cat = "GA", 
+                            race = TRUE, age.limits = c(15, 65),
+                            age.breaks = c(20, 30, 40, 50, 60),
+                            init.hiv.prev = c(0.17,0.1,0.13))
+
+#Specify age categories: (0-20], (20, 30], . . . (60, 100]
+netparams1 <- build_netparams(epistats = epistats8, smooth.main.dur = TRUE)
+
```

Comment:
please add these all into unit tests as well that checks for consistency of input and expected values